src/etc/man/cargo-doc.1

```diff
@@ -73,7 +73,7 @@ Build for the target triple.
 .RE
 .TP
 .B \-\-manifest\-path \f[I]PATH\f[]
-Path to the manifest to compile.
+Path to the manifestsrc/etc/man/cargo-doc.1 to compile.
```

Comment:
This should be `Path to the manifest (Cargo.toml)`? 

Also, I'd just simplified all instances to `path to the Cargo.toml`: the flag name already mentions manifest, no need to repeat that :)

Reply:
... When copy/paste goes wrong. I wanted to do that, but didn’t want to take too many liberties. Will fix.